CONTRIBUTING.md

```diff
@@ -125,34 +144,39 @@ Chokidar version:
 
 ### Changelog
 
-The `CHANGELOG.md` file must be updated for any new components or changes that you add. If unsure of which release your changes will go to, you can add a placeholder version and date (on the top):
+The `CHANGELOG.md` file must be updated for any new components or changes that you add. At the top of the file, create an "unreleased" section with a placeholder date (if not already present).
+
+<details>
```

Comment:
It is not that long, maybe no need to collapse.

Reply:
I just kept the same as css repo. It helps with the readability of the whole document